andela/mmdp_site
# MMDP-357 Fix login form style inconsistencies

**What does this PR do?**
Fix the inconsistent font style on Content Management System login page.

**Description of Task to be completed?**
The font style should be consistent across all the pages in the Content Management System.

**How should this be manually tested?**

1. Fetch all branches - `git fetch --all`.
2. Checkout to this branch - `git checkout bg-fix-login-styles-MMDP-357`.
3. Run the application - `yarn dev-server`.
4. If you're logged in log out. If not you will be on the login page.
5. Check to confirm that the login page has the correct font i.e. `Muli`.
6. Check also that there is a margin between the `Sign in` button and the `Forgot password` link.

**Any background context you want to provide?**

Although not part of the story requirements I have moved the styles file out of the component directory to the `assets` directory. This is in adherence to the team convections.

**What are the relevant Jira issues?**

[MMDP-357](https://viisaus.atlassian.net/browse/MMDP-357)

**Screenshots (if appropriate)**
#### Before
![screenshot 2019-03-05 at 11 53 27](https://user-images.githubusercontent.com/8180548/53792711-56016e00-3f3d-11e9-91bc-d923de3f64a5.png)

#### After
![screenshot 2019-03-05 at 11 57 09](https://user-images.githubusercontent.com/8180548/53792911-db851e00-3f3d-11e9-937f-91e103a6bc55.png)

**Questions:**
N/A
